app/templates/content/index-crown-hosting.html

```diff
@@ -96,7 +96,7 @@ <h2 class='summary-item-heading'>
         </ul>
       </div>
       <div class="govuk-grid-column-one-third">
-        <h2>
+        <h2 class="govuk-heading-m">
```

Comment:
While we're sorting out typography on this page, can we do something about the `<strong>` tag around `Crown Commercial Service` on line 103? I don't like the positioning to the right, either 😹  

My ideal solution for this would be to move that block into the main column (underneath the 'read more' section, which goes after the contact section on mobile view), and have something like this: 

```
<h2 class="govuk-heading-m">Crown Hosting contact</h2>
<p class="govuk-body">Email <a class="govuk-link" href="mailto:crownhosting@crowncommercial.gov.uk">crownhosting@crowncommercial.gov.uk</a> if you need support buying or selling Crown Hosting services.</p>
```
![crown-hosting](https://user-images.githubusercontent.com/3492540/85834604-9668a900-b78b-11ea-936d-60af49f4647f.png)

However I realise this is getting a little out of scope!

@NoraGDS What do you think?

Reply:
Ah good idea. I'm happy to do that in this PR.

Reply:
Why do we need the email address on the page twice? 🤔

Reply:
Good question!

Reply:
The second paragraph from the top can be deleted. It duplicates the information in the contact section, which looks clearer in the new position as well.

Reply:
Thanks @NoraGDS !